mainwindow.cpp

```diff
@@ -6,6 +6,7 @@ MainWindow::MainWindow(QWidget *parent)
     , ui(new Ui::MainWindow)
 {
     ui->setupUi(this);
+    player1->intitialize(ui);
```

Comment:
why player2 not being initialize?

Reply:
Me miss that
Me sorry
Me got it updated in the next commit

Reply:
done